bin/pic-configure

```diff
@@ -165,6 +165,15 @@ if [ -f "$extension_param/cmakeFlags" ] ; then
         echo "       Is the file executable? (chmod u+x cmakeFlags)" >&2
         exit 2
     fi
+    # save the cmakeFlags setup number for future reference
+    if [ -f "$extension_param/cmakeFlagsSetup" ] ; then
+      rm $extension_param/cmakeFlagsSetup
+    fi
+    touch $extension_param/cmakeFlagsSetup
+    echo "Last configured cmakeFlags setup was: $cmakeFlagsNr" > $extension_param/cmakeFlagsSetup
+    echo -e "This was the content of cmakeFlags:\n"
+    cat  $extension_param/cmakeFlags >> $extension_param/cmakeFlagsSetup
```

Comment:
Do not copy the file in, copy the setup string only. The file is already available 

should be
```
echo $cmake_flags >> $extension_param/cmakeFlagsSetup
```